lib/Client.js

```diff
@@ -566,7 +552,10 @@ class Client extends EventEmitter {
                 }
             }
         }
-        command.default_permission = command.defaultPermission;
+
+        command.default_member_permissions = command.defaultMemberPermissions;
+        command.dm_permission = command.DMPermission;
```

Comment:
```suggestion
        command.dm_permission = command.dmPermission;
```
keys should be camel cased